src/js/static.js

```diff
@@ -22,8 +22,8 @@ gpii.express.router["static"].init = function (that) {
     that.staticHandler = express["static"](that.options.content);
 };
 
-gpii.express.router["static"].route = function (that, req, res) {
-    that.staticHandler(req, res);
+gpii.express.router["static"].route = function (that, req, res, next) {
+    that.staticHandler(req, res, next);
```

Comment:
I believe that this was only possible because we had a "bad actor" from the express 3.x days that had "router-like middleware".  I cannot reproduce it with the unpatched code and any combination of normal `gpii.express.middleware` and `gpii.express.router` modules.